lms/djangoapps/instructor_task/tasks.py

```diff
@@ -176,3 +177,14 @@ def get_ora2_responses(entry_id, xmodule_instance_args):
     action_name = ugettext_noop('generated')
     task_fn = partial(push_ora2_responses_to_s3, xmodule_instance_args)
     return run_main_task(entry_id, task_fn, action_name)
+
+
+# pylint: disable=not-callable
+@task(base=BaseInstructorTask, routing_key=settings.ORA2_RESPONSES_DOWNLOAD_ROUTING_KEY)
```

Comment:
I don't think it creates another celery queue, it just lets you specify which one to use. I think it would be better to have its own settings instead of using ORA2, they all get set to HIGH_MEM_QUEUE in lms/envs/common.py anyways